ocdata/utils/vasp.py

```diff
@@ -175,17 +187,17 @@ def write_vasp_input_files(atoms, outdir='.', vasp_flags=None):
                     Defaults to '.'
         vasp_flags  A dictionary of settings we want to pass to the `Vasp`
                     calculator. Defaults to a standerd set of values if `None`
-    '''
+    """
     if vasp_flags is None:  # Immutable default
         vasp_flags = VASP_FLAGS.copy()
 
-    atoms, vasp_flags = _clean_up_inputs(atoms, vasp_flags)
+    atoms, vasp_flags = _clean_up_inputs(atoms, vasp_flags.copy())
     calc = Vasp(directory=outdir, **vasp_flags)
     calc.write_input(atoms)
 
 
-def xml_to_tuples(xml='vasprun.xml'):
-    '''
+def xml_to_tuples(xml="vasprun.xml"):
+    """
```

Comment:
not used - remove

Reply:
Updated in https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset/commit/ac78be51e6108b9d69c071386844617d715ff4ab